auth/gcloud/aio/auth/iam.py

```diff
@@ -70,7 +72,8 @@ async def get_public_key(self, key_id: Optional[str] = None,
 
         resp = await s.get(url=url, headers=headers, timeout=timeout)
 
-        return await resp.json()
+        data: Dict[str, str] = await resp.json()
```

Comment:
That's not the first time I see parsed json as `Dict[str, str]`. Why isn't it `Dict[str, Any]`?

Reply:
According to the [API docs](https://cloud.google.com/iam/docs/reference/rest/v1/projects.serviceAccounts.keys/get), the response body should look like [this](https://cloud.google.com/iam/docs/reference/rest/v1/projects.serviceAccounts.keys#ServiceAccountKey), which just so happens to be `Dict[str, str]`. I figured more specific is better where possible!